ignite/pkg/cosmosaccount/cosmosaccount.go

```diff
@@ -133,12 +139,18 @@ func (a Account) Address(accPrefix string) string {
 		accPrefix = AccountPrefixCosmos
 	}
 
-	return toBench32(accPrefix, a.Info.GetPubKey().Address())
+	// TODO: handle error
+	pk, _ := a.Record.GetPubKey()
```

Comment:
I am not sure of all the implication of changing the function signature to return the error, but I would definitely return it if possible. If so we would have to update the blog client tutorial in PR #2667 because is using this method.

Regarding the [error](https://github.com/cosmos/cosmos-sdk/blob/main/crypto/keyring/record.go#L66) returned by `GetPubKey`, it looks "esoteric" so maybe it would be nice to wrap it in a custom error with a more user friendly message.